src/main/java/org/javarosa/core/model/instance/TreeReference.java

```diff
@@ -676,4 +676,10 @@ public TreeReference removePredicates() {
         }
         return predicateless;
     }
+
+    public TreeReference removePredicates(int i) {
```

Comment:
I'm mirroring what's done at `removePredicates`, which also returns a mutated clone :mutant_emoji: 

Do you think we should change the naming here? I wouldn't be able to just rename `removePredicates()` because it's public, though (I'd have to deprecate it)

Reply:
Ahh, I didn't see it was an overload of an existing method. And this is only needed for `Scenario.getRef`. Let's keep it for now and try to remove it when addressing #422.